### ISSUE TITLE
Manually editing the hooks/pre-commit file

### ISSUE THREAD

**Issue by keeganstreet** [nlf/precommit-hook#62]
Hi,

When I install this package, it creates a `hooks/pre-commit` file.

I need to manually edit this file and add the following line to allow the hook to be run from Source Tree as well as the command line.
```
export PATH=/usr/local/bin:$PATH
```

Editing this file in vim is fine, and allows my hooks to be run from Source Tree. However if I edit the file in Sublime Text on Mac, the hooks no longer run, even on the command line. Instead I see this error:

```
sed: 1: "s/
$//": unterminated substitute pattern
EXPECTED value GOT EOF
failed parsing /Users/kestreet/myproject/package.json.. exiting
```

If I compare the original `hooks/pre-commit` file generated by this package with the file I have edited in Sublime Text, the original file has "mixed line endings" and the edited file has Unix (LF) line endings.

Do you know why this package is breaking when the `hooks/pre-commit` file has Unix (LF) line endings?

**Comment by functionobject** [comment]
This seems to still be an issue and is probably currently caused by https://github.com/nlf/git-validate/issues/34.

**Comment by booleanbetrayal** [comment]
Also experiencing this issue!

**Comment by nlf** [comment]
looking in to this, i need a less fragile way of handling newlines

**Comment by codeStryke** [comment]
also experiencing this issue :+1: 
any workarounds?

**Comment by nlf** [comment]
just published version 2.0.5 of git-validate that i'm hoping resolves this, can someone who was having issues check it out?

**Comment by nlf** [comment]
marking this as closed. reopen if someone continues having issues on the latest version (2.1.0)
